content/en/docs/distributions/gke/deploy/deploy-cli.md

```diff
@@ -101,63 +68,118 @@ purpose. No tools will assume they actually exists in your terminal environment.
      ```bash
      GO111MODULE=on go get github.com/mikefarah/yq/v3
      ```
-
-   * If you don't have [Go](https://golang.org) installed,  follow the
+    
+    * If you don't have [Go](https://golang.org) installed,  follow the
      instructions in the yq repository to
-     [install yq v3](https://github.com/mikefarah/yq#install).
+     [install yq v3](https://github.com/mikefarah/yq#install). For example:
+
+        ```bash
+        sudo wget https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_amd64 -O /usr/bin/yq && sudo chmod +x /usr/bin/yq
+        yq --version
+        # yq version 3.4.1
+        ```
    
    **Note:** The Kubeflow deployment process is not compatible with yq v4 or later.
 
-1. Follow the instructions from [Preparing to install Anthos Service Mesh](https://cloud.google.com/service-mesh/docs/archive/1.4/docs/gke-install-new-cluster#preparing_to_install_anthos_service_mesh) to install `istioctl`.
+1. Install jq https://stedolan.github.io/jq/, for example, we can run the following command on Linux: 
 
-    Note, the `istioctl` downloaded from above instructions is specific to Anthos Service Mesh. It is different from the `istioctl` you can download on <https://istio.io/>.
+    ```bash
+    sudo apt install jq
+    ```
 
-## Prepare your environment
+### Fetch kubeflow/gcp-blueprints and upstream packages
 
-1. Log in. You only need to run this command once:
+1. If you have already installed Management cluster, you have `kubeflow/gcp-blueprints` locally. You just need to run `cd kubeflow` to access Kubeflow cluster manifests. Otherwise, you can run the following commands:
 
     ```bash
-    gcloud auth login
+    # Check out Kubeflow v1.3.0 blueprints
+    git clone https://github.com/kubeflow/gcp-blueprints.git 
+    git checkout tags/v1.3.0
+
+    # Visit Kubeflow cluster related manifests
+    cd gcp-blueprints/kubeflow
```

Comment:
nit: we do not have instructions to set up `git` CLI, shall we document `kpt pkg get` as an alternative?

Reply:
Adding the kpt pkg option, `git` seems to be a common enough tool where we don't need to provide installation guide, and we have provided the alternative here.